users/models.py

```diff
@@ -14,12 +15,15 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     first_name = models.TextField(max_length=50, blank=False, verbose_name='first name')
     last_name = models.TextField(max_length=50, blank=False, verbose_name='last name')
+    gender = models.TextField(max_length=20, blank=False, default='male', verbose_name='gender')
```

Comment:
User can enter `asd123` as gender, which is not valid. Let's check `choices` option of text field.